wiki/Manual.md

```diff
@@ -2561,17 +2561,9 @@ Obviously, it takes some time for the exchange to collect the first-order data a
 
 The latency (or how much time is needed by the exchange API for calculating the secondary data) depends on how fast the exchange engine is, so it is exchange-specific. Top exchange engines will usually return and update fresh last-minute OHLCV candles and tickers at a very fast rate. Some exchanges might do it in regular intervals like once a second or once in a few seconds. Slow exchange engines might take minutes to update the secondary statistical information, their APIs might return the current most recent OHLCV candle a few minutes late.
 
-If your strategy depends on the fresh last-minute most recent data you don't want to build it based on tickers or OHLCVs received from the exchange. Tickers and exchanges' OHLCVs are only suitable for display purposes, or for simple trading strategies for hour-timeframes or day-timeframes that are less susceptible to latency.
```

Comment:
this is inaccurate statement, you can normally use any normal exchange's minute OHLCV (binance,okx...) data  and  you can hardly see any important difference if you manually build bars. yes, there might be some exceptions during high volatility or server overload, but absolutely in most cases the difference is not that much degree as say that "OHLCV can only be used for display purposes" and "usable only for high-timeframe strategies"